### PR TITLE
EDU-413 Redesign search results table

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-413

### CODE REVIEW

File: src/components/pages/search.less [hunk 1]
```diff
@@ -48,3 +48,34 @@
...
 }
+
+.SearchPage-cellTagsExpandLink {
+  font-size: 0.9em;
+  display: inline-block;
+  margin-left: 0.2em;
+  height: 20px;
+  padding: 3px;
+  margin: 3px;
+  margin-left: 1em;
```
Comment (adrianaluchian) [inline src/components/pages/search.less:59]: The margins here are a bit confusing, `margin-left` is defined twice and there's a mix of margins in `px` and `em` 🤔

File: src/components/items-expander.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const showExpander = items.length > initialItemCount + 1;
+
+  return (
+    <div className={className}>
+      {items.filter((_tag, index) => isExpanded || (showExpander && index < initialItemCount)).map(renderItem)}
+      {showExpander && isExpanded && (
+        <a className={expandLinkClassName} onClick={() => setIsExpanded(false)}>{t('collapse')}</a>
+      )}
+      {showExpander && !isExpanded && (
+        <a className={expandLinkClassName} onClick={() => setIsExpanded(true)}>{t('expand', { count: items.length - 10 })}</a>
```
Comment (adrianaluchian) [inline src/components/items-expander.js:18]: Here `10` could be replaced with `initialItemCount`, if I get the purpose of this variable right.

Comment (ahelmberger) [reply src/components/items-expander.js]: Thanks for spotting that

File: src/components/pages/search.less (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/search.less]: I replaced it anyway :-)

Comment (adrianaluchian) [reply src/components/pages/search.less]: The first `margin-left` could still be removed, since you set `margin` afterwards, right?

Comment (ahelmberger) [reply src/components/pages/search.less]: OMG, yes, of course. Removed it!

**Pull request merged**